locales/frFR.lua

```diff
@@ -603,32 +602,32 @@ for key,value in pairs({
 		--TODO: ["Expansion Pre"] = "Pre",
 		--TODO: ["Expansion Features"] = "EF",
 		--TODO: [GROUP_FINDER] = "D&R",	-- ["Dungeons & Raids"] = "D&R"
-		--TODO: ["The Burning Crusade"] = "BC",
-		--TODO: ["Burning Crusade"] = "BC",
-		--TODO: ["The BC"] = "BC",
-		--TODO: ["Wrath of the Lich King"] = "WotLK",
-		--TODO: ["Cataclysm"] = "Cata",
-		--TODO: ["Mists of Pandaria"] = "MoP",
-		--TODO: ["Warlords of Draenor"] = "WoD",
-		--TODO: ["Battle for Azeroth"] = "BFA",
-		--TODO: ["The Shadowlands"] = "SL",
-		--TODO: ["Shadowlands"] = "SL",
-		--TODO: ["Player vs Player"] = "PvP",
+		["The Burning Crusade"] = "BC",
+		["Burning Crusade"] = "BC",
+		["The BC"] = "BC",
+		["Wrath of the Lich King"] = "WotLK",
+		["Cataclysm"] = "Cata",
+		["Mists of Pandaria"] = "MoP",
+		["Warlords of Draenor"] = "WoD",
+		["Battle for Azeroth"] = "BFA",
+		["The Shadowlands"] = "SL",
+		["Shadowlands"] = "SL",
+		["Player vs Player"] = "JcJ",
 		--TODO: ["Raid Finder"] = "LFR",
 		--TODO: ["Looking For Raid"] = "LFR",
-		--TODO: ["Normal"] = "N",
-		--TODO: ["Heroic"] = "H",
-		--TODO: ["Mythic"] = "M",
+		["Normal"] = "N",
+		["Heroic"] = "H",
+		["Mythic"] = "M",
 		["Ny’alotha, la cité en éveil"] = "Ny’alotha",	-- ["Ny'alotha, the Waking City"] = "Ny'alotha"
 		["Tazavesh, le marché dissimulé"] = "Tazavesh",	-- ["Tazavesh, the Veiled Market"] = "Tazavesh"
-		--TODO: ["10 Player"] = "10M",
-		--TODO: ["10 Player (Heroic)"] = "10M (H)",
-		--TODO: ["25 Player"] = "25M",
-		--TODO: ["25 Player (Heroic)"] = "25M (H)",
+		-["10 Player"] = "10J",
+		["10 Player (Heroic)"] = "10J (H)",
+		["25 Player"] = "25J",
+		["25 Player (Heroic)"] = "25J (H)",
 		--TODO: ["Emissary Quests"] = "Emissary",
 		--TODO: [TRACKER_HEADER_WORLD_QUESTS] = "WQ",	-- ["World Quests"] = "WQ"
-		--TODO: ["WoW Anniversary"] = "Anniversary",
-		--TODO: ["Covenant:"] = "Cov:",
+		["WoW Anniversary"] = "Anniversaire",
+		["Covenant:"] = "Cov :",
```

Comment:
This `ABBREVIATIONS` table is needed to shorten tooltips in ATT, so instead of "Dungeons & Raids -> Wrath of the Lich King -> RaidName -> ..." it would be just "D&R -> WotLK -> RaidName -> ..."

To make it work the left side of the table (full name) should be EXACTLY how it looks in game.

So for example I suppose "WoW Anniversary" is "WoW Anniversaire" in Frech and you'd need to edit the left side as well, not only the short name on the right. Same for all expansion names and other entries you've edited.